tests/e2e/containers/config.go

```diff
@@ -24,7 +24,7 @@ const (
 	// It should be uploaded to Docker Hub. OSMOSIS_E2E_SKIP_UPGRADE should be unset
 	// for this functionality to be used.
 	previousVersionOsmoRepository = "osmolabs/osmosis-dev"
-	previousVersionOsmoTag        = "16.0"
+	previousVersionOsmoTag        = "16.0-alpine"
```

Comment:
Note to reviewer: I am tired of not being able to run a shell within our docker containers, so I changed the image